sunspot/spec/api/indexer/child_documents_spec.rb

```diff
@@ -0,0 +1,18 @@
+require File.expand_path('spec_helper', File.dirname(__FILE__))
+
+if Sunspot::Util.child_documents_supported?
```

Comment:
maybe better to use this technique? 
https://relishapp.com/rspec/rspec-core/docs/pending-and-skipped-examples/skip-examples#skipping-using-metadata